app/DynamicsAdapter/DynamicsAdapter.Web/MatchFound/MatchFoundController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace DynamicsAdapter.Web.MatchFound
+{
+    public class MatchFoundResponse
+    { 
+        public string SearchRequestId { get; set; }
+        public string SearchResult { get; set; }
+    }
+
+
+    [Route("api/[controller]")]
```

Comment:
can we go just `"[controller]"` dotnet core 3 is getting rid of the api path

Reply:
sure